react-cra/src/pages/WebComponents.tsx

```diff
@@ -273,6 +273,27 @@ export default function WebComponents() {
           </button>
         </sdds-modal>
       </div>
+
+      <div>
+        <div className="sdds-headline-02 sdds-u-pb1">Radio Button</div>
+        <sdds-radio-button
+            name="rb-example"
+            value="option1"
+            radio-id="option-1"
+            checked
```

Comment:
No warning on this one?

Reply:
![Screenshot 2023-03-10 at 14 17 43](https://user-images.githubusercontent.com/17004063/224325937-afe18324-64ab-419e-bb53-c90a0260e64e.png)
 Just a value missing warning, like with any prop that is boolean...